LayoutTests/imported/w3c/web-platform-tests/dom/observable/tentative/observable-constructor.any-expected.txt

```diff
@@ -1,38 +1,50 @@
-FAIL Subscription reports errors that are pushed after subscriber is closed by completion Can't find variable: Observable
-FAIL Errors thrown by initializer function after subscriber is closed by completion are reported Can't find variable: Observable
-FAIL Errors thrown by initializer function after subscriber is closed by error are reported Can't find variable: Observable
-FAIL Errors pushed by initializer function after subscriber is closed by error are reported Can't find variable: Observable
-FAIL Subscriber#complete() cannot re-entrantly invoke itself Can't find variable: Observable
-FAIL Subscriber#error() cannot re-entrantly invoke itself Can't find variable: Observable
-FAIL Unsubscription lifecycle Can't find variable: Observable
-FAIL Aborting a subscription should stop emitting values Can't find variable: Observable
-FAIL Calling subscribe should never throw an error synchronously, initializer throws error Can't find variable: Observable
-FAIL Calling subscribe should never throw an error synchronously, subscriber pushes error Can't find variable: Observable
-FAIL Teardown should be called when subscription is aborted Can't find variable: Observable
-FAIL Teardowns should be called when subscription is closed by completion Can't find variable: Observable
-FAIL Teardowns should be called when subscription is closed by subscriber pushing an error Can't find variable: Observable
-FAIL Teardowns should be called when subscription is closed by subscriber throwing error Can't find variable: Observable
-FAIL Teardowns should be called synchronously during addTeardown() if the subscription is inactive Can't find variable: Observable
+PASS Observable constructor
+PASS subscribe() can be called with no arguments
+PASS Subscriber interface is not constructible
+PASS Subscribe with just a function as the next handler
+PASS Observable constructor calls initializer on subscribe
+PASS Observable error path called synchronously
+PASS Subscriber must have receiver
+PASS Subscriber next & error must recieve argument
+PASS Subscriber complete() will set active to false, and abort signal
+PASS Subscriber active is readonly
+PASS Subscriber signal is readonly
+PASS Observable should error if initializer throws
+PASS Subscription is inactive after complete()
+PASS Subscription is inactive after error()
+PASS Subscription is inactive when aborted signal is passed in
+PASS Subscriber#signal is not the same AbortSignal as the one passed into `subscribe()`
+PASS Subscription does not emit values after completion
+PASS Subscription does not emit values after error
+PASS Completing or nexting a subscriber after an error does nothing
+FAIL Errors pushed to the subscriber that are not handled by the subscription are reported to the global assert_equals: Error message matches expected "Uncaught Error: custom error" but got "Error: custom error"
```

Comment:
I think this failure is something that needs to be addressed in WPTs. WebKit doesn't prepend `Uncaught` in re-raised exceptions, generally. The WPTs assume it will (largely because they were written alongside the Chrome implementation). I believe this is also the case in Firefox.